Data-Profiles.md

```diff
@@ -41,5 +41,57 @@ Yearly 26kb / data source
 
 ![Data Source Profiles RRA Edit](images/data-source-profiles-edit2.png)
 
+### Changing data profile averages 
+
+The Cacti gui allows you to change the averages on a specific Data source profile
```

Comment:
GUI should be written as GUI not gui because it is an abbreviation same as RRD (Round Robin Database) 👍

Reply:
fixed